### DISCUSSION TITLE
Change layout in specific process

### DISCUSSION THREAD

**Discussion by ghost** [DreymaR/BigBagKbdTrixPKL#87] [Q&A] [answered]
Is it possible to automatically switch to another keyboard layout for specific process names? Something like the suspending apps function, but instead of suspending EPKL, changing to another layout. And this may be a niche thing, but if possible, I would like to know if, only in specific processes, it is possible to switch to determined layouts and then disable it with another key. I understand this may be a feature too isolated, but if there is a possibility it would be nice. Apart from that, everything has been running very smoothly!

**Comment by DreymaR** [comment, marked as answer]
I have not implemented anything like that, and I think it may be too niche as you say.

However, you could play around with some other tricks. You can use the normal multi-layout functionality of EPKL with a simple-to-use switch key (default is Ctrl+Shift+2). You can have EPKL auto-suspend itself on a certain system layout and let that layout be the one you want (maybe you have to make it in MSKLC then; see the Other\MSKLC folder of the EPKL repo).

**Comment by ghost** [comment]
For those who are interested, the function 
`changeLayout( layoutname) { 						; Rerun EPKL with a specified layout`

allows you to do what I asked before.
It may be useful for those who want to automate layout switching
